addons/spreadsheet_dashboard_sale_timesheet/data/dashboards.xml

```diff
@@ -4,6 +4,8 @@
     <record id="spreadsheet_dashboard_timesheet" model="spreadsheet.dashboard">
         <field name="name">Timesheets</field>
         <field name="spreadsheet_binary_data" type="base64" file="spreadsheet_dashboard_sale_timesheet/data/files/timesheet_dashboard.json"/>
+        <field name="main_data_model_ids" eval="[(4, ref('analytic.model_account_analytic_line'))]"/>
```

Comment:
this isn't enough, timesheet always creates 2 entries. would adding project_project  solve this ?

Reply:
There is always a default project created by default called "Internal"
<img width="1466" alt="image" src="https://github.com/user-attachments/assets/60039121-14ed-461d-9be3-73f4b9950b18">

Reply:
There is no project created when `project` is installed, and when I install all spreadsheet_dashboard_* modules I still do not have any project. Do you have the xmlid of this record ? I would like to see where it comes from

Reply:
![image](https://github.com/user-attachments/assets/a9644ee1-5e43-4918-9643-e67b2686909b)

Reply:
It comes from `hr_timesheet`.
So IMO two options:
- Add `project.project` in main_model_ids, as at least it will works for database that only install project and dashboard for exemple (best effort)
- Remove this sample dashboard if we consider the first option not enough.

WDYT @VincentSchippefilt ?

Reply:
option 1: let's add projet.project